examples/large_files/test/tests.ts

```diff
@@ -1,233 +1,90 @@
-                        4n
-                    );
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized get hash status from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.get_hash_status('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized get hash from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.get_file_hash('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        {
-            name: 'Unauthorized clear file and info from actor',
-            test: async () => {
-                const actor = await createActor(
-                    getCanisterId('backend'),
-                    unauthorizedUser
-                );
-                try {
-                    await actor.clear_file_and_info('assets/test0B');
-                } catch (err: any) {
-                    return {
-                        Ok: err.message.includes(
-                            'Not Authorized: only controllers of this canister may call this method'
-                        )
-                    };
-                }
-                return { Ok: false };
-            }
-        },
-        ...generateStandardFileTests('Upload', origin),
-        {
-            name: 'redeploy',
-            prep: async () => {
-                await generateTestFileOfSize(1, 'KiB');
-                await generateTestFileOfSize(10, 'KiB');
-                await generateTestFileOfSize(100, 'KiB');
-                execSync(`dfx deploy --upgrade-unchanged`, {
+    return () => {
+        beforeAll(async () => {
```

Comment:
The manual folder needs to be cleared out as well

Reply:
done